frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -277,6 +280,81 @@ TransparentThemeNoBackgroundScroll.decorators = [
   ScrollDecorator,
 ];
 
+// Other components compatibility test
+export function ComponentCompatibility() {
+  return (
+    <div>
+      <Tooltip
+        label={
+          <Text size="sm" c="var(--mb-color-text-primary)">
+            Label
+          </Text>
+        }
+        opened
+      >
+        <Card withBorder display="inline-block">
+          Mantine Tooltip
+        </Card>
+      </Tooltip>
+      <LegacyTooltip
+        tooltip={
+          <Text size="sm" c="var(--mb-color-text-primary)">
+            Label
+          </Text>
+        }
+        isOpen
+      >
+        <Card withBorder display="inline-block">
+          Legacy Tooltip
+        </Card>
+      </LegacyTooltip>
+      <Popover withArrow shadow="md" opened>
+        <Popover.Target>
+          <Card withBorder display="inline-block">
+            Mantine Popover
+          </Card>
+        </Popover.Target>
+        <Popover.Dropdown>
+          <Text size="sm" c="var(--mb-color-text-primary)">
+            Dropdown
+          </Text>
+        </Popover.Dropdown>
+      </Popover>
+      <TippyPopoverWithTrigger
+        isInitiallyVisible
+        triggerContent={
+          <Card withBorder display="inline-block">
+            Tippy Popover
+          </Card>
+        }
+        popoverContent={
+          <Text size="sm" c="var(--mb-color-text-primary)">
+            Dropdown
+          </Text>
+        }
+      />
+      <PopoverWithTrigger
+        isInitiallyOpen
+        triggerElement={
+          <Card withBorder display="inline-block">
+            Legacy Popover
+          </Card>
+        }
+      >
+        {() =>
+          function Inner({ maxHeight, ...props }: { maxHeight: number }) {
```

Comment:
If we don't intercept `maxHeight` it will be passed to `Text` causing runtime error because `maxHeight` isn't an HTML attribute, causing Loki to fail.